src/problem_data.jl

```diff
@@ -149,3 +151,5 @@ function load_problem!(pb::ProblemData{T},
 
     return pb
 end
+
+is_continuous(pb::ProblemData) = all(pb.var_types .== CONTINUOUS)
```

Comment:
We could have a "is_continuous" flag within `ProblemData`, so that this needs to run only once when loading the problem.

Reply:
I was worried about that getting out of sync, but if we don't allow incrementally adding variables that shouldn't be an issue...

Reply:
Done.